tests/func/test_version.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from dvc.command.version import CmdVersion
+from dvc.cli import parse_args
+
+
+def test_run():
+    cmd = CmdVersion(parse_args(["version"]))
+    ret = cmd.run_cmd()
+    assert ret == 0
+
+
+def test_info(caplog):
+    check_data = ["DVC version", "Python version", "Platform"]
+    cmd = CmdVersion(parse_args(["version"]))
+    cmd.run()
+    info_dict = {}
+
+    for record in caplog.records:
```

Comment:
@algomaster99 since you are only using one record, there's no need to iterate over all the caplog records, you can use `caplog.text`.